src/app/related-search/related-search.component.spec.ts

```diff
@@ -77,7 +77,18 @@ describe('RelatedSearchComponent', () => {
     fixture.detectChanges();
   });
 
-  it('should create', () => {
+  it('should create an instance', () => {
     expect(component).toBeTruthy();
   });
+
+  it('should display results if any, along with right keyword', () => {
+    let compiled = fixture.debugElement.nativeElement;
+    if (component.results.length > 0) {
```

Comment:
In the jasmine official tutorial, there was no mention of initializing, he said you can access values of the component using its name, that is why I did this.